components/eam/src/physics/cam/physics_utils.F90

```diff
@@ -32,6 +33,53 @@ module physics_utils
 
 #endif
 
-    contains
+contains
+
+  pure function calculate_drymmr_from_wetmmr(ncols, wetmmr, qv_wet) result (drymmr)
+    !Compute drymmr for any wetmmr constituent using wet water vapor mixing ratio(qv_wet)
```

Comment:
since common usage of "mixing ratio" is what we call drymmr and wetmmr is really the specific humidity, maybe edit comment to "Compute drymmr (mass of water divided by mass of dry air; commonly known as mixing ratio) for any wetmmr constituent (mass of water divided by mass of dry air plus water vapor; see specific humidity) using qv_wet (mass of vapor divided by mass of dry air plus vapor).